tests/test_iter.py

```diff
@@ -47,29 +46,34 @@ def test_trie_next_prev_using_fixtures(fixture_name, fixture):
         assert nxt == iterator.next(point)
 
 
-@given(random=strategies.randoms())
-@settings(max_examples=10, deadline=500)
-def test_iter(random):
-    trie, contents = make_random_trie(random)
+@given(random_trie_strategy())
+@settings(max_examples=200)
```

Comment:
Shouldn't we keep the deadline here and in test_iter_all?

Reply:
Since the test is faster, it completes within the default deadline of 200ms now. Might as well go with the tighter deadline now, I suppose.